### PR TITLE
Release 1.8.2 (master)

### PR DESCRIPTION
**Pull request by marinafrydenb**
### Version 1.8.2

* Added a more thorough description on `paymentUrl` and `completeUrl` in all relevant
  payments.
* Updated rderItem to be required in all requests and responses.
* Clean up of Swish Payments.
* Removed `pageStripDown` in Vipps Redirect.
* Updated Card Payments Direct to have a correct integration
  flow.
* Updated the main page to be more welcoming.
* Updated information on Merchant Authenticated Consumer.
* Added docker files, updated remote references and workflow file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,18 +18,26 @@ well as [Swedbank Pay Open Source Development Guidelines][dev-guidelines].
...
 3. To install the [Ruby Gems][gems] this web site requires, you first need to
    install [Bundler][bundler].
-4. Once Ruby and Bundler is in place, type `bundle install` inside the root
+4. Install [Graphviz][graphviz], this will require Java.
+5. Once Ruby ,Bundler, and Graphviz, is in place, type `bundle install` inside the root
    folder of this repository.
-5. Run `bundle exec jekyll serve` to start the website.
-6. Open `http://localhost:4000` in a browser.
-7. In Visual Studio Code, install the following plugins:
+6. Run `bundle exec jekyll serve` to start the website.
```
Comment (payex-dev) [inline README.md:38]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 37
- Added 34
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607&bid=17552091)

File: _includes/field-description-termsofserviceurl.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    unless `generateReferenceToken` or `generateRecurrenceToken` is also submitted
+    in the request. This is the Merchants, not the Swedbank Pay Terms of
+    Service. **HTTPS is a requirement**.
+{%- endcapture -%}
+{{- termsofserviceurl | strip_newlines -}}
+
+{%- comment -%}
+The dashes in the Liquid code tags remove output whitespace. More on that here:
+
+https://shopify.github.io/liquid/basics/whitespace/
```
Comment (payex-dev) [inline _includes/field-description-termsofserviceurl.md:13]: Codacy Issue found: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: payments/card/direct.md [hunk 3]
```diff
@@ -28,53 +28,71 @@ sidebar:
...
+- The action taken next is the `direct-authorization` operation that is returned
+  in the first request. You `POST` the payer's card data to the URL in the
+  [`direct-authorization`][authorization] operation.
+- If the issuer requires 3-D Secure authentication, you will then receive an
+  operation called `redirect-authentication`. You must redirect the payer to
+  this URL to let them authenticate against the issuer's 3-D Secure page.
+  - When the 3-D Secure flow is completed, the payer will be redirected back to
+      the URL provided in `completeUrl` or `cancelUrl`, depending on the actions
+      performed by the payer.
+  - If the issuer does not require 3-D Secure authentication, the payment will
```
Comment (payex-dev) [inline payments/card/direct.md:62]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: README.md [hunk 4]
```diff
@@ -18,18 +18,26 @@ well as [Swedbank Pay Open Source Development Guidelines][dev-guidelines].
...
+
+### Manual install
 1. [Clone this repository][cloning].
 2. Jekyll is written in [Ruby][ruby], so you'll need to download and install
    that. If you're installing on Windows, choose setup with DevKit.
 3. To install the [Ruby Gems][gems] this web site requires, you first need to
    install [Bundler][bundler].
-4. Once Ruby and Bundler is in place, type `bundle install` inside the root
+4. Install [Graphviz][graphviz], this will require Java.
+5. Once Ruby ,Bundler, and Graphviz, is in place, type `bundle install` inside the root
```
Comment (payex-dev) [inline README.md:36]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: payments/card/direct.md [hunk 5]
```diff
@@ -28,53 +28,71 @@ sidebar:
...
+
+- The payer places an order and you make a `Purchase` request towards Swedbank
+  Pay with the gathered payment information.
+- The action taken next is the `direct-authorization` operation that is returned
+  in the first request. You `POST` the payer's card data to the URL in the
+  [`direct-authorization`][authorization] operation.
+- If the issuer requires 3-D Secure authentication, you will then receive an
+  operation called `redirect-authentication`. You must redirect the payer to
+  this URL to let them authenticate against the issuer's 3-D Secure page.
+  - When the 3-D Secure flow is completed, the payer will be redirected back to
```
Comment (payex-dev) [inline payments/card/direct.md:59]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: README.md [hunk 6]
```diff
@@ -18,18 +18,26 @@ well as [Swedbank Pay Open Source Development Guidelines][dev-guidelines].
...
    folder of this repository.
-5. Run `bundle exec jekyll serve` to start the website.
-6. Open `http://localhost:4000` in a browser.
-7. In Visual Studio Code, install the following plugins:
+6. Run `bundle exec jekyll serve` to start the website.
+7. Open `http://localhost:4000` in a browser.
```
Comment (payex-dev) [inline README.md:39]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: payments/card/direct.md [hunk 7]
```diff
@@ -28,53 +28,71 @@ sidebar:
...
-  received in the first step, which will return the purchase result.
+## Payment flow
+
+Below is a quick stepwise summary of how the Direct Card Payment scenario works.
+
+- The payer places an order and you make a `Purchase` request towards Swedbank
+  Pay with the gathered payment information.
+- The action taken next is the `direct-authorization` operation that is returned
+  in the first request. You `POST` the payer's card data to the URL in the
+  [`direct-authorization`][authorization] operation.
```
Comment (payex-dev) [inline payments/card/direct.md:55]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: payments/card/direct.md [hunk 8]
```diff
@@ -28,53 +28,71 @@ sidebar:
...
+- If the issuer requires 3-D Secure authentication, you will then receive an
+  operation called `redirect-authentication`. You must redirect the payer to
+  this URL to let them authenticate against the issuer's 3-D Secure page.
+  - When the 3-D Secure flow is completed, the payer will be redirected back to
+      the URL provided in `completeUrl` or `cancelUrl`, depending on the actions
+      performed by the payer.
```
Comment (payex-dev) [inline payments/card/direct.md:61]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: README.md (hunk 6 above)
Comment (payex-dev) [inline README.md:39]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: README.md [hunk 9]
```diff
@@ -18,18 +18,26 @@ well as [Swedbank Pay Open Source Development Guidelines][dev-guidelines].
...
+2. Run `docker-compose up`
+
+### Manual install
 1. [Clone this repository][cloning].
 2. Jekyll is written in [Ruby][ruby], so you'll need to download and install
    that. If you're installing on Windows, choose setup with DevKit.
 3. To install the [Ruby Gems][gems] this web site requires, you first need to
    install [Bundler][bundler].
-4. Once Ruby and Bundler is in place, type `bundle install` inside the root
+4. Install [Graphviz][graphviz], this will require Java.
```
Comment (payex-dev) [inline README.md:35]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: README.md [hunk 10]
```diff
@@ -18,18 +18,26 @@ well as [Swedbank Pay Open Source Development Guidelines][dev-guidelines].
...
 
 To view this website, browse to [developer.swedbankpay.com][swp-dp]. If you'd
-like to host it locally on your computer, you need to do the following:
+like to host it locally on your computer, you have to options;
+[docker][docker], or [manual][manual-install] install.
 
+### Docker
+
+1. Install [docker][docker-install] for your operating system.
+2. Run `docker-compose up`
```
Comment (payex-dev) [inline README.md:27]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: _includes/field-description-orderitems.md [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+    The array of items being purchased with the order.
+    Used to print on invoices if the payer chooses to pay with invoice, among
+    other things. `orderItems` is required in all requests.
+    In `capture` requests it should only contain the items to be captured from the order.
+{%- endcapture -%}
+{{- orderitems | strip_newlines -}}
+{%- comment -%}
+The dashes in the Liquid code tags remove output whitespace. More on that here:
+
+https://shopify.github.io/liquid/basics/whitespace/
```
Comment (payex-dev) [inline _includes/field-description-orderitems.md:11]: Codacy Issue found: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: payments/card/direct.md [hunk 12]
```diff
@@ -28,53 +28,71 @@ sidebar:
...
+  - If the issuer does not require 3-D Secure authentication, the payment will
+      already be `Completed` after performing the `direct-authorization`
+      request. Note that `Completed` just indicates that the payment is in a
+      final state; the financial transaction could be either OK or failed.
+- Finally you make a `GET` request towards Swedbank Pay with the `id` of the
```
Comment (payex-dev) [inline payments/card/direct.md:66]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: _includes/callback-reference.md [hunk 13]
```diff
@@ -7,11 +7,19 @@ When a change or update from the back-end system are made on a payment or
...
-  you receive information about what happened with the payment.
+Providing a `callbackUrl` in `POST` requests is **mandatory**. Below we provide
+three example scenarios of why this is important:
+
+1. If the payer closes the payment window, the merchant will never know what
+    happened to the payment if `callbackUrl` is not implemented.
+2. If the payer stops up in a payment app such as Vipps or Swish, the payer
+    will never come back to the merchant. This means that the merchant won't know what
+    happened to the payment unless `callbackUrl` is implemented.
+3. If a payer experiences a network error or something else happens that
```
Comment (payex-dev) [inline _includes/callback-reference.md:18]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: README.md [hunk 14]
```diff
@@ -18,18 +18,26 @@ well as [Swedbank Pay Open Source Development Guidelines][dev-guidelines].
 ## Usage
 
 To view this website, browse to [developer.swedbankpay.com][swp-dp]. If you'd
-like to host it locally on your computer, you need to do the following:
+like to host it locally on your computer, you have to options;
+[docker][docker], or [manual][manual-install] install.
 
+### Docker
+
+1. Install [docker][docker-install] for your operating system.
```
Comment (payex-dev) [inline README.md:26]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: _includes/payment-url.md [hunk 15]
```diff
@@ -29,3 +30,46 @@ URL of the Payment stored.
...
+| `onPaymentFailed`            | will call registrered endpoint if provided. If not subscribed to will redirect to conpleteUrl                                                       |
+| `onPaymentCanceled`          | will be called if the payment is aborted by the consumer or triggered by you with the Abort command on the payment                                  |
+| `onPaymentTransactionFailed` | this will trigger if one payment attempt failed. Standard functionality are to let the consumer be able to retry the payment from the payment page. |
+
+When implementing the Seamless View flow into a WebView in your mobile app, you
+should use a [custom scheme][custom-scheme] or [Universal Link][universal-link]
+in the `paymentUrl` for handling automatic switching between your app and the
+payment app on the mobile device.
+
+[custom-scheme]: https://developer.apple.com/documentation/uikit/inter-process_communication/allowing_apps_and_websites_to_link_to_your_content/defining_a_custom_url_scheme_for_your_app
```
Comment (payex-dev) [inline _includes/payment-url.md:73]: Codacy Issue found: [[final-definition] Move definitions to the end of the file (after the node at line `75`)](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: README.md (hunk 1 above)
Comment (payex-dev) [inline README.md:38]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: payments/card/direct.md [hunk 16]
```diff
@@ -231,104 +299,104 @@ Content-Type: application/json
...
-* If the payment shown above is done as a two-phase (Authorization), you will
+- If the payment shown above is done as a two-phase (Authorization), you will
   need to implement the [Capture][capture] and [Cancel][cancel] requests.
-* **Abort:** It is possible to [abort a payment][abort] if the payment has no
+- **Abort:** It is possible to [abort a payment][abort] if the payment has no
   successful transactions.
-* For `reversals`, you will need to implement the [Reversal][reversal] request.
-* **Callback from Swedbank Pay:** Whenever changes to the payment occur a
+- For `reversals`, you will need to implement the [Reversal][reversal] request.
+- **Callback from Swedbank Pay:** Whenever changes to the payment occur a
```
Comment (payex-dev) [inline payments/card/direct.md:399]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: _includes/callback-reference.md [hunk 17]
```diff
@@ -7,11 +7,19 @@ When a change or update from the back-end system are made on a payment or
...
 
-* Setting a `callbackUrl` in the HTTP `POST` API is optional, but highly
-  recommended. If a payer closes the browser window, a network error or
-  something else happens that prevents the payer from being redirect from
-  Swedbank Pay back to the merchant website, the callback is what ensures that
-  you receive information about what happened with the payment.
+Providing a `callbackUrl` in `POST` requests is **mandatory**. Below we provide
+three example scenarios of why this is important:
+
+1. If the payer closes the payment window, the merchant will never know what
```
Comment (payex-dev) [inline _includes/callback-reference.md:13]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: payments/swish/index.md [hunk 18]
```diff
@@ -98,6 +104,7 @@ Swish App.
 [payex-admin-portal]: https://admin.payex.com/psp/login/
 [payex-mailto]: mailto:sales@payex.com
 [SEB-swish]: https://seb.se/foretag/digitala-tjanster/swish-handel
+[settlement-and-reconciliation]: /payments/swish/other-features#settlement-and-reconciliation
```
Comment (payex-dev) [inline payments/swish/index.md:107]: Codacy Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: _includes/payment-url.md [hunk 19]
```diff
@@ -29,3 +30,46 @@ URL of the Payment stored.
...
+[universal-link]: https://developer.apple.com/ios/universal-links/
```
Comment (payex-dev) [inline _includes/payment-url.md:74]: Codacy Issue found: [[final-definition] Move definitions to the end of the file (after the node at line `75`)](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: _includes/callback-reference.md [hunk 20]
```diff
@@ -7,11 +7,19 @@ When a change or update from the back-end system are made on a payment or
...
+    happened to the payment if `callbackUrl` is not implemented.
+2. If the payer stops up in a payment app such as Vipps or Swish, the payer
```
Comment (payex-dev) [inline _includes/callback-reference.md:15]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: _includes/callback-reference.md [hunk 21]
```diff
@@ -7,11 +7,19 @@ When a change or update from the back-end system are made on a payment or
...
+    will never come back to the merchant. This means that the merchant won't know what
+    happened to the payment unless `callbackUrl` is implemented.
```
Comment (payex-dev) [inline _includes/callback-reference.md:17]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: payments/card/direct.md [hunk 22]
```diff
@@ -28,53 +28,71 @@ sidebar:
...
-  merchant/webshop site.
-* Finally you make a `GET` request towards Swedbank Pay with the `paymentID`
-  received in the first step, which will return the purchase result.
+## Payment flow
+
+Below is a quick stepwise summary of how the Direct Card Payment scenario works.
+
+- The payer places an order and you make a `Purchase` request towards Swedbank
+  Pay with the gathered payment information.
+- The action taken next is the `direct-authorization` operation that is returned
```
Comment (payex-dev) [inline payments/card/direct.md:53]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: payments/card/direct.md [hunk 23]
```diff
@@ -231,104 +299,104 @@ Content-Type: application/json
...
-  deactivate Merchant
 ```
 
 ## Options after posting a purchase payment
 
-* If the payment shown above is done as a two-phase (Authorization), you will
+- If the payment shown above is done as a two-phase (Authorization), you will
   need to implement the [Capture][capture] and [Cancel][cancel] requests.
-* **Abort:** It is possible to [abort a payment][abort] if the payment has no
+- **Abort:** It is possible to [abort a payment][abort] if the payment has no
```
Comment (payex-dev) [inline payments/card/direct.md:396]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 37
- Added 34
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607&bid=17552091)

File: _includes/callback-reference.md (hunk 20 above)
Comment (payex-dev) [inline _includes/callback-reference.md:15]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: payments/card/direct.md [hunk 24]
```diff
@@ -28,53 +28,71 @@ sidebar:
...
+## Payment flow
+
+Below is a quick stepwise summary of how the Direct Card Payment scenario works.
+
+- The payer places an order and you make a `Purchase` request towards Swedbank
+  Pay with the gathered payment information.
+- The action taken next is the `direct-authorization` operation that is returned
+  in the first request. You `POST` the payer's card data to the URL in the
+  [`direct-authorization`][authorization] operation.
+- If the issuer requires 3-D Secure authentication, you will then receive an
```
Comment (payex-dev) [inline payments/card/direct.md:56]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: _includes/payment-url.md (hunk 15 above)
Comment (payex-dev) [inline _includes/payment-url.md:73]: Codacy Issue found: [[final-definition] Move definitions to the end of the file (after the node at line `75`)](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: payments/card/direct.md [hunk 25]
```diff
@@ -28,53 +28,71 @@ sidebar:
...
+  operation called `redirect-authentication`. You must redirect the payer to
+  this URL to let them authenticate against the issuer's 3-D Secure page.
+  - When the 3-D Secure flow is completed, the payer will be redirected back to
+      the URL provided in `completeUrl` or `cancelUrl`, depending on the actions
+      performed by the payer.
+  - If the issuer does not require 3-D Secure authentication, the payment will
+      already be `Completed` after performing the `direct-authorization`
+      request. Note that `Completed` just indicates that the payment is in a
+      final state; the financial transaction could be either OK or failed.
```
Comment (payex-dev) [inline payments/card/direct.md:65]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: README.md (hunk 9 above)
Comment (payex-dev) [inline README.md:35]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: README.md (hunk 14 above)
Comment (payex-dev) [inline README.md:26]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: _includes/callback-reference.md (hunk 13 above)
Comment (payex-dev) [inline _includes/callback-reference.md:18]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: payments/card/direct.md (hunk 8 above)
Comment (payex-dev) [inline payments/card/direct.md:61]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: README.md [hunk 26]
```diff
@@ -18,18 +18,26 @@ well as [Swedbank Pay Open Source Development Guidelines][dev-guidelines].
...
-4. Once Ruby and Bundler is in place, type `bundle install` inside the root
+4. Install [Graphviz][graphviz], this will require Java.
+5. Once Ruby ,Bundler, and Graphviz, is in place, type `bundle install` inside the root
    folder of this repository.
-5. Run `bundle exec jekyll serve` to start the website.
-6. Open `http://localhost:4000` in a browser.
-7. In Visual Studio Code, install the following plugins:
+6. Run `bundle exec jekyll serve` to start the website.
+7. Open `http://localhost:4000` in a browser.
+8. In Visual Studio Code, install the following plugins:
```
Comment (payex-dev) [inline README.md:40]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: README.md (hunk 6 above)
Comment (payex-dev) [inline README.md:39]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: payments/card/direct.md (hunk 7 above)
Comment (payex-dev) [inline payments/card/direct.md:55]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: payments/card/direct.md (hunk 5 above)
Comment (payex-dev) [inline payments/card/direct.md:59]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: payments/card/direct.md (hunk 22 above)
Comment (payex-dev) [inline payments/card/direct.md:53]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: payments/card/direct.md (hunk 23 above)
Comment (payex-dev) [inline payments/card/direct.md:396]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: payments/card/direct.md [hunk 27]
```diff
@@ -231,104 +299,104 @@ Content-Type: application/json
...
-  SwedbankPay-->>+Merchant: rel: redirect-authorization
-  deactivate SwedbankPay
-  Merchant-->>Payer: display purchase result
-  deactivate Merchant
 ```
 
 ## Options after posting a purchase payment
 
-* If the payment shown above is done as a two-phase (Authorization), you will
+- If the payment shown above is done as a two-phase (Authorization), you will
```
Comment (payex-dev) [inline payments/card/direct.md:394]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: _includes/callback-reference.md (hunk 21 above)
Comment (payex-dev) [inline _includes/callback-reference.md:17]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: payments/card/direct.md [hunk 28]
```diff
@@ -28,53 +28,71 @@ sidebar:
...
-* When the payment is completed, the payer needs to be redirected back to your
-  merchant/webshop site.
-* Finally you make a `GET` request towards Swedbank Pay with the `paymentID`
-  received in the first step, which will return the purchase result.
+## Payment flow
+
+Below is a quick stepwise summary of how the Direct Card Payment scenario works.
+
+- The payer places an order and you make a `Purchase` request towards Swedbank
+  Pay with the gathered payment information.
```
Comment (payex-dev) [inline payments/card/direct.md:52]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: payments/card/direct.md (hunk 3 above)
Comment (payex-dev) [inline payments/card/direct.md:62]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: _includes/field-description-termsofserviceurl.md (hunk 2 above)
Comment (payex-dev) [inline _includes/field-description-termsofserviceurl.md:13]: Codacy Issue found: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: payments/card/direct.md (hunk 12 above)
Comment (payex-dev) [inline payments/card/direct.md:66]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: README.md [hunk 29]
```diff
@@ -43,7 +51,7 @@ like to host it locally on your computer, you need to do the following:
    * `supperchong.pretty-json` to format selected JSON snippets in code
      examples.
    * `sissel.shopify-liquid` for syntax highlighting of [Liquid][liquid].
-8. Also in Visual Studio Code, [set up a ruler at 80 characters][vsc-ruler]
+9. Also in Visual Studio Code, [set up a ruler at 80 characters][vsc-ruler]
```
Comment (payex-dev) [inline README.md:54]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: README.md (hunk 9 above)
Comment (payex-dev) [inline README.md:35]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: payments/swish/index.md (hunk 18 above)
Comment (payex-dev) [inline payments/swish/index.md:107]: Codacy Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: _includes/callback-reference.md (hunk 17 above)
Comment (payex-dev) [inline _includes/callback-reference.md:13]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: README.md (hunk 6 above)
Comment (payex-dev) [inline README.md:39]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)

File: resources/release-notes.md [hunk 30]
```diff
@@ -30,6 +30,20 @@ published on this page." %}
 body="The version numbers used in headers on this page refers to the version of
 this very documentation, not to a version of any APIs described by it." %}
 
+## April 2020
+
+### Version 1.8.2
+
+* Added a more thorough description on `paymentUrl` and `completeUrl` in all relevant
+  payments.
+* Updated òrderItem`to be required in all requests and responses.
```
Comment (asbjornu) [inline resources/release-notes.md:39]:
```suggestion
* Updated `orderItems` to be required in all requests and responses.
```

File: payments/card/direct.md [hunk 31]
```diff
@@ -110,35 +138,33 @@ Content-Type: application/json
...
-            "description": "Test transaction",
-            "payeeReference": "1583846025",
+            "description": "Books & Ink",
+            "payeeReference": "cyrusLibrary1586291679",
             "isOperational": false,
             "operations": [
                 {
                     "method": "PATCH",
-                    "href": "https://api.stage.payex.com/psp/creditcard/payments/{{ page.payment_id }}/authorizations/{{ page.transaction_id }}",
+                    "href": "https://api.internaltest.payex.com/psp/creditcard/payments/{{ page.payment_id }}/authorizations/{{ page.transaction_id }}",
```
Comment (NullableInt) [inline payments/card/direct.md:167]:
```suggestion
                    "href": "{{ page.api_url}}/psp/creditcard/payments/{{ page.payment_id }}/authorizations/{{ page.transaction_id }}",
```

**Pull request merged**